Package.swift

```diff
@@ -16,6 +16,8 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-crypto.git", from: "3.0.0"),
         .package(url: "https://github.com/apple/swift-certificates.git", branch: "main"),
         .package(url: "https://github.com/attaswift/BigInt.git", from: "5.3.0"),
+        .package(url: "https://github.com/apple/swift-nio.git", from: "2.0.0"),
```

Comment:
Do we need this dependency if we've copied everything in? It would be good not to have these as dependencies if possible

Reply:
Ended up removing the test so we don't need that anymore